android/app/src/main/kotlin/com/jabook/app/jabook/audio/processors/SpeechEnhancer.kt

```
@@ -42,21 +42,22 @@ public class SpeechEnhancer : AudioProcessor {
+    private var highPassAlpha = 0.0f
+    private var highPassPrev = FloatArray(0)
 
-    // Peak EQ state (simplified - using gain multiplier for target frequency range)
-    private val peakEqGainLinear = 10.0.pow((peakEqGainDb / 20.0f).toDouble()).toFloat()
+    // Peak EQ biquad state (frequency-selective 3kHz boost), per channel
+    private var peakEqB0 = 0f
+    private var peakEqB1 = 0f
+    private var peakEqB2 = 0f
+    private var peakEqA1 = 0f
+    private var peakEqA2 = 0f
+    private lateinit var peakEqState: Array<BiquadState>
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)